File: PromoStudio.Common/Models/VoiceActor.cs
using System;
using PromoStudio.Common.Enumerations;

namespace PromoStudio.Common.Models
{
    public class VoiceActor
    {
        public int pk_VoiceActorId { get; set; }
        public sbyte fk_VoiceActorStatusId { get; set; }
        public string UserName { get; set; }
        public string FullName { get; set; }
        public string Description { get; set; }
        public string SampleFilePath { get; set; }
        public string PhotoFilePath { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime? DateUpdated { get; set; }

        public VoiceActorStatus Status
        {
            get { return (VoiceActorStatus) fk_VoiceActorStatusId; }
            set { fk_VoiceActorStatusId = (sbyte) value; }
        }

        public dynamic ToPoco()
        {
            return new
            {
                pk_VoiceActorId,
                fk_VoiceActorStatusId,
                UserName,
                FullName,
                Biography = Description,
                SampleFilePath,
                PhotoFilePath,
                DateCreated,
                DateUpdated
            };
        }
    }
}

File: PromoStudio.Web/Scripts/models/voiceActor.js
/// <reference path="../vsdoc/require.js" />
/// <reference path="../vsdoc/knockout-2.3.0.debug.js" />

"use strict";

define(["knockout"], function (ko) {
    function ctor (data) {
        var self = this;
        data = data || {};

        self.pk_VoiceActorId = ko.observable(data.pk_VoiceActorId || null);
        self.fk_VoiceActorStatusId = ko.observable(data.fk_VoiceActorStatusId || null);
        self.UserName = ko.observable(data.UserName || null);
        self.FullName = ko.observable(data.FullName || null);
        self.Description = ko.observable(data.Description || null);
        self.SampleFilePath = ko.observable(data.SampleFilePath || null);
        self.PhotoFilePath = ko.observable(data.PhotoFilePath || null);
        self.DateCreated = ko.observable(data.DateCreated || null);
        self.DateUpdated = ko.observable(data.DateUpdated || null);
    }

    ctor.prototype.toJSON = function () {
        var copy = ko.toJS(this);
        // remove any unneeded properties
        delete copy.SampleFilePath;
        delete copy.PhotoFilePath;
        delete copy.DateCreated;
        delete copy.DateUpdated;

        return copy;
    };
    return ctor;
});
